src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -173,8 +180,15 @@ private void setListMethodFields(
     PageStreamingConfig pageStreaming = context.getMethodConfig().getPageStreaming();
     TypeRef resourceType = pageStreaming.getResourcesField().getType();
     String resourceTypeName = context.getTypeTable().getAndSaveNicknameForElementType(resourceType);
+    String resourceFieldName =
+        context.getNamer().getFieldGetFunctionName(pageStreaming.getResourcesField());
```

Comment:
If this is a field name, it shouldn't be acquired using a namer function which produces a string formatted as a function name. (In C#, function names are formatted differently than fields - functions are upper camel, fields are lower camel)